extension/src/test/suite/plots/util.ts

```diff
@@ -61,6 +63,17 @@ export const buildPlots = async (
   )
 
   mockHasCheckpoints(expShow)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const experimentsModel: ExperimentsModel = (experiments as any).experiments
+  experimentsModel.setSelected([
+    { id: 'workspace' },
+    { id: 'main' },
+    { id: 'exp-e7a67' },
+    { id: 'test-branch' },
+    { id: 'exp-83425' },
+    { id: 'exp-f13bca' }
+  ] as Experiment[])
```

Comment:
[F] Had to do this to keep the test fixture looking the same way for all of the plots tests. This should be updated later.